pkg/cert/request.go

```diff
@@ -65,7 +64,7 @@ func NewServerCertificateManager(kubeClient clientset.Interface, namespace, secr
 			// authenticate itself to a TLS client.
 			certificates.UsageServerAuth,
 		},
-		SignerName:          certificates.LegacyUnknownSignerName,
+		SignerName:          "kubernetes.io/legacy-unknown",
```

Comment:
Hard coding this string since `LegacyUnknownSignerName` is no longer available in `v1`, as per https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers.